src/utils.js

```diff
@@ -1,11 +1,19 @@
 // jarvis, just another rudimentary verbal interface shell
-// converts 'hello "John Doe"' to ['hello', 'John, Doe']
 const fs = require("fs");
 const path = require("path");
+
+// checks whether the given argument is in the format of a stringified JSON
+const isJson = (argument) => {
+  return /<({.+})>/.test(argument);
```

Comment:
how is this json?

Reply:
Stringified  JSON is wrapped by the `<` `>` to make it more robust while tokenizing. That's the reason for the above format validation to check whether the input format is in the JSON format.